src/GitVersion.Core/VersionCalculation/VersionSearchStrategies/TaggedCommitVersionStrategy.cs

```diff
@@ -41,6 +43,7 @@ private IEnumerable<BaseVersion> GetBaseVersionsInternal(EffectiveBranchConfigur
 
         var label = configuration.Value.GetBranchSpecificLabel(Context.CurrentBranch.Name, null);
 
+        SemanticVersion semanticVersionTreshold = new(0, 0, 0);
         List<SemanticVersionWithTag> alternativeSemanticVersionsWithTag = [];
         foreach (var semanticVersionWithTag in taggedSemanticVersions)
```

Comment:
I think `taggedSemanticVersion` reads better than `semanticVersionWithTag` and is also more consistent with `taggedSemanticVersions`.

```suggestion
        foreach (var taggedSemanticVersion in taggedSemanticVersions)
```